synapse/pipeline.yml

```diff
@@ -400,7 +388,7 @@ steps:
       - "bash /bootstrap.sh synapse"
     plugins:
       - docker#v3.7.0:
-          image: "matrixdotorg/sytest-synapse:py38"
+          image: "matrixdotorg/sytest-synapse:bullseye"
```

Comment:
the py38 and py37 images are no longer maintained, so I'm updating them while I'm here.